### PR TITLE
Newinternal API itests Part 1

### PR DESCRIPTION
**Pull request by Matthewrbowker**

### CODE REVIEW

File: tests/includes/API/ApiExceptionTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
```
Comment (stwalkerster) [inline tests/includes/API/ApiExceptionTest.php]: This file was also added by you in be87a2f - you may want to rebase onto the most recent newinternal commit and resolve the conflicts.

File: tests/includes/API/ApiExceptionTest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
```
Comment (stwalkerster) [inline tests/includes/API/ApiExceptionTest.php]: We have git for finding out who created a file :)

File: tests/includes/API/ApiExceptionTest.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	private $message;
+	private $exception;
+
+	public function setUp() {
+		$this->message = "This is a test ApiException.";
+
+		try {
+			throw new ApiException($this->message);
+		}
+		catch(ApiException $e) {
```
Comment (stwalkerster) [inline tests/includes/API/ApiExceptionTest.php]: Also, general code style comments 
- opening braces go on the next line for methods and classes, but on the same line for control structures
- on this line specifically, a space between the `catch` and the `(` please

File: tests/includes/API/ApiExceptionTest.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ * Date: 5/21/16
+ * Time: 22:35
+ */
+
+use \Waca\API\ApiException;
+
+class ApiExceptionTest extends PHPUnit_Framework_TestCase
+{
+	private $message;
+	private $exception;
```
Comment (stwalkerster) [inline tests/includes/API/ApiExceptionTest.php]: `/** @var ApiException */` will help code analysis tools work here if you stick it before `private $exception;`

File: tests/includes/API/Actions/CountActionTest.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+class CountActionTest extends PHPUnit_Framework_TestCase
+{
+	private $count;
+
+	public function setUp() {
+		$count = new CountAction();
+
+		$this->count = $count;
+
+		$this->assertInstanceOf(CountAction::class, $count);
```
Comment (stwalkerster) [inline tests/includes/API/Actions/CountActionTest.php]: I'm fairly sure this is guaranteed to pass by the PHP runtime.

(It is PHP though, so who knows...)

**Comment by Matthewrbowker** [comment]
I have made the changes requested, and will push them with #7, you are welcome to close this duplicated work (that I didn't realize I had duplicated)

**Pull request closed without merging**